### ISSUE TITLE
SC problematic for Closed functionality: SC 2.1.1 Keyboard

### ISSUE THREAD

**Issue by pday1** [w3c/wcag2ict#319]
For ease of review, here is content pulled from the latest Editor's draft to show the original WCAG SC language and "Applying SC to Non-Web..." alongside the proposed new content for "SC problematic for Closed...".

The latest WCAG2ICT editor's draft can be found at https://w3c.github.io/wcag2ict/
(Note this does not contain content that is in pull requests that are open and not yet approved)

**WCAG 2.2 Text for 2.1.1 Keyboard**

All [functionality](https://www.w3.org/TR/WCAG22/#dfn-functionality) of the content is operable through a [keyboard interface](https://www.w3.org/TR/WCAG22/#dfn-keyboard-interface) without requiring specific timings for individual keystrokes, except where the underlying function requires input that depends on the path of the user's movement and not just the endpoints.

NOTE 1: This exception relates to the underlying function, not the input technique. For example, if using handwriting to enter text, the input technique (handwriting) requires path-dependent input but the underlying function (text input) does not.

NOTE 2: This does not forbid and should not discourage providing mouse input or other input methods in addition to keyboard operation.

**Content from Applying SC**
https://w3c.github.io/wcag2ict/#applying-sc-2-1-1-keyboard-to-non-web-documents-and-software

> This applies directly as written, and as described in [Intent from Understanding Success Criterion 2.1.1](https://www.w3.org/WAI/WCAG22/Understanding/keyboard#intent).
> NOTE 1
> This does not imply that software always needs to directly support a keyboard or “keyboard interface”. Nor does it imply that software always needs to provide a soft keyboard. Underlying platform software may provide device independent input services to applications that enable operation via a keyboard. Software that supports operation via such platform device independent services would be operable by a keyboard and would comply.
>
> NOTE 2
> See also the [Comments on Closed Functionality](https://w3c.github.io/wcag2ict/#comments-on-closed-functionality).

**Latest proposed content for SC problematic for Closed functionality section**

See [2.1.1 Google Doc](https://docs.google.com/document/d/13ZzHeccY_0V79OhUFAP2Uw5-JvgkcpHEuForc_MVWjI/edit?usp=sharing) for discussion and previously proposed content. 

**Option 1: Current text in editor’s draft**

> 2.1.1 Keyboard—Requires operation via a keyboard interface which allows alternative input devices. When a product with closed functionality either does not have a standard keyboard or one cannot be connected, it would need an alternate way to access all functionality that does not require accurate pointing, path-based movements, or specific timings.
 
**Option 5: Compromise position - Phil**

> Requires operation via a keyboard interface which also allows for alternative input devices. When a product with closed functionality does not have a standard keyboard or an alternative input device cannot be connected, it may not be possible to satisfy this success criterion. It may be possible to address some user needs (such as offering input methods that support users with low vision, without vision, or limited manual dexterity).

**Comment by pday1** [comment]
@maryjom Issue created for ease of reference in next survey to TF. Content taken from extra meeting on [Friday 08 March 2024](https://www.w3.org/2024/03/08-wcag2ict-minutes.html).

**Comment by maryjom** [comment]
Consensus reached on 4 April: [Incorporate option 5 for 2.1.1 Keyboard into the SC Problematic for Closed Functionality section, as-is.](https://www.w3.org/2024/04/04-wcag2ict-minutes#r01).

See PR #337.
